src/main/java/com/google/sps/data/SessionContext.java

```diff
@@ -32,21 +45,25 @@ public SessionContext(UserRepository userRepository, UserService userService) {
   * method that returns the current logged in User or null if
   * no user is logged in
   */
-  public User getLoggedInUser() {
+  public User getLoggedInUser() throws Exception {
```

Comment:
Is throws Exception needed in L48 or L62?

Reply:
+1 I don't think this method throws an exception.

Reply:
Done